packages/ts-autocomplete/package.json

```diff
@@ -54,7 +54,7 @@
   "dependencies": {
     "debug": "^4.4.0",
     "lodash": "^4.17.21",
-    "typescript": "^5.0.4"
+    "typescript": "^5.3.3"
```

Comment:
Same version as in mongosh.

Reply:
Here in the repo the package-lock points to 5.8.2 anyway, feel free to also bump it to that in mongosh too (not a blocker for this PR really, the version here in package.json doesn't really matter anyway)